packages/sil-voting-app/src/store/proposals/actions.ts

```diff
@@ -61,6 +63,7 @@ export async function getProposals(governor: Governor): Promise<ProposalsState>
         },
         result: proposal.result,
         vote: await getVote(proposal),
+        hasStarted: creationDate < new Date(),
```

Comment:
There is no operator overloading in JavaScript. So I think `<` between `Date` objects does not what you think it does. You can use `.getTime()` on both sides and compare milliseconds as integers

Reply:
A couple of StackOverflow answers I've seen suggest this actually works fine. (You can use `<` and `>`, but not `==`.)

https://stackoverflow.com/a/13285569

Reply:
Alright, 👍 then